README.md

```diff
@@ -1,2 +1,65 @@
 # spdx-python-model
 Generated Python code for SPDX Spec version 3
+
+All bindings in this repository are generated using
+[shacl2code](https://github.com/JPEWdev/shacl2code) at the time the package is
+built.
+
+**NOTE:** The bindings are pretty low level, intended for more directly
+manipulating SPDX files. While they are fully functions, they lack higher level
+helper functions that may be useful for creating SPDX documents. If you want a
+higher level approach, please see the
+[SPDX Python Tools](https://github.com/spdx/tools-python) (however this repo
+doesn't yet support SPDX 3)
+
+## Usage
+
+Each version of the SPDX spec has a module named `v{MAJOR}_{MINOR}_{MICRO}`
+that contains the bindings for that version under the `spdx_python_model` top
+level. For example:
+
+```python
+import spdx_python_model
+
+p = spdx_python_model.v3_0_1
```

Comment:
```suggestion
m = spdx_python_model.v3_0_1
```

Suggest using a different variable from the person since they represent different objects

Reply:
This was actually just a typo. I'll fix it.